packages/core/src/rules/ajv.ts

```diff
@@ -14,20 +11,17 @@ export function releaseAjvInstance() {
 function getAjv(resolve: ResolveFn<any>, disallowAdditionalProperties: boolean) {
   if (!ajvInstance) {
     ajvInstance = new Ajv({
-      schemaId: 'auto',
+      schemaId: '$id',
```

Comment:
@andriyl review all the new and deprecated options and adjust it.

E.g. we do need `discriminator: true`